components/ContentCard/ContentCard.js

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+import Link from "next/link";
+
+const ContentCard = ({imgSrc, imgAlt, contentType, title, caption, link}) => {
+  return (
+    <div>
+      <Link href={link}>
+        <a>
+          <div className="flex flex-col justify-end w-64 md:w-96 h-24 md:h-48 ">
+            <Image className="rounded-t-2xl" src={imgSrc} alt={imgAlt}></Image>
+          </div>
+          <div className="flex flex-col justify-end w-64 md:w-96 h-48 bg-blue-50 rounded-b-2xl">
+            <h3 className="text-base md:text-lg text-blue-500 px-5">
+              {contentType}
+            </h3>
+            <h4 className="text-xl md: text-2xl font-semibold px-5">{title}</h4>
+            <p className="text-base md:text-lg px-5">
+              {caption.slice(0, 1000) + "......"}
```

Comment:
Hi Negin. Looks good but I wonder if this might be a neater (and more consistent) way of achieving what you're looking for:
[Tailwind CSS text overflow](https://tailwindcss.com/docs/text-overflow)

Reply:
Thank you, I had no idea Tailwind has that. I'll have a look and apply it to my code